.github/workflows/main.yml

```diff
@@ -47,6 +47,22 @@ jobs:
     - name: Test with tox
       run: tox -e ${{ matrix.tox-env }}
 
+  xarray:
+    needs: pypi
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tox-env: [xarray]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install tox
+        run: pip install tox
+      - name: Test with tox against xarray@master
+        run: tox -e ${{ matrix.tox-env }}
+
```

Comment:
@cehbrecht before I was waiting for the other tests before running the xarray one, rather than running in parallel, so I think that's why it would take longer. I've changed to `py37-xarray` now, but can change it to do something else if you prefer.